### PR TITLE
changing landing colors & backgrounds

### PR DESCRIPTION
**Pull request by midnight-sl**

### CODE REVIEW

File: web/src/screens/LandingScreen/LandingScreen.pcss [hunk 1]
```diff
@@ -227,10 +236,11 @@
...
 
   &__register {
-    background-image: url(assets/images/landing/TradingUIFullBG.png);
-    background-position: right center;
-    background-repeat: no-repeat;
-    background-size: 80%;
+    background: url(assets/images/landing/TradingRegister.svg) no-repeat center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
```
Comment (josadcha) [inline web/src/screens/LandingScreen/LandingScreen.pcss]: actually pcss generates this values after compilation, so you can remove that

Comment (midnight-sl) [reply web/src/screens/LandingScreen/LandingScreen.pcss]: good to know this, thanks
will remove it now

File: web/src/components/TickerTable/index.tsx [hunk 2]
```diff
@@ -1,34 +1,50 @@
...
+                    <td className="currency">
+                        {currency.icon_url ? (
+                            <span className="cr-crypto-icon currency__icon">
+                                <img alt={currency.name.toUpperCase()} src={currency.icon_url} />
+                            </span>
+                        ) : (<CryptoIcon className="currency__icon" code={currency.name.toUpperCase()} />)}
+                        <div className="currency__names">
+                           {market && market.name ?  
+                           <span>
+                               <span className="currency__names__bold">{market.name.split('/')[0]}</span>
```
Comment (josadcha) [inline web/src/components/TickerTable/index.tsx:43]: you can also get that data by `market.base_unit`

File: web/src/components/TickerTable/index.tsx [hunk 3]
```diff
@@ -1,34 +1,50 @@
...
+                               <span className="currency__names__light">/{market.name.split('/')[1]}</span>
```
Comment (josadcha) [inline web/src/components/TickerTable/index.tsx:44]: and` market.quote_unit` here

**Pull request closed without merging**